dashboard/common/util/__init__.py

```diff
@@ -0,0 +1,40 @@
+# Some utility functions used by other classes in this project
+
+import django
+
+def get_course_name_from_id(course_id):
```

Comment:
I feel we should log the function call as `logger.info(get_course_name_from_id.__name__)` as it is easy for debugging. I don't know it is a best practice.

Reply:
I commented on #227 that we should just probably look into using decorators instead. I can add this as a new issue.

Reply:
new issue is better. Let's get this checked in first.